### PR TITLE
NAS-121156 / 23.10 / Add vaultwarden to community train

### PR DESCRIPTION
**Pull request by stavros-k**
- Added only environment variables that required for the setup.
  - Extra variables can be passed via the additional environment variables section or from the GUI (which takes precedence) of vaultwarden
- Added certificate option for HTTPS, but also added in description + readme, that is recommended to use external reverse proxy.(https://github.com/dani-garcia/vaultwarden/wiki/Enabling-HTTPS)

### CODE REVIEW

File: library/ix-dev/community/vaultwarden/values.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+image:
+  repository: vaultwarden/server
+  pullPolicy: IfNotPresent
+  tag: "1.27.0"
```
Comment (stavros-k) [inline library/ix-dev/community/vaultwarden/values.yaml:4]:
```suggestion
  repository: ghcr.io/dani-garcia/vaultwarden
  pullPolicy: IfNotPresent
  tag: "1.28.0"
```

File: library/ix-dev/community/vaultwarden/values.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+image:
+  repository: ghcr.io/dani-garcia/vaultwarden
+  pullPolicy: IfNotPresent
+  tag: "1.28.0"
```
Comment (stavros-k) [inline library/ix-dev/community/vaultwarden/values.yaml:4]:
```suggestion
  repository: vaultwarden/server
  pullPolicy: IfNotPresent
  tag: "1.27.0"
```

**Comment by stavros-k** [comment]
Depends on #1071

**Review by Qubad786 — COMMENTED** [review]
You are missing `upgrade_info` file as well, please check other apps and add that file too..

**Review by Qubad786 — APPROVED** [review]

**Pull request merged**